### DISCUSSION TITLE
HTML book: is there any native way to read what the active footnote is?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#10632] [Q&A]
### Description

Hi,
if I open this link to footnote 2 https://quarto.org/docs/authoring/markdown-basics.html#fnref2, I have the page below (after clicking on the note number).
I'm thinking of a site with `citations-hover: false`.

Is there any way to read what the active footnote is? It would be very convenient to have a visual way to figure out which one I should read.

If no, I would open a feature request.

Thank you

![image](https://github.com/user-attachments/assets/528798c2-8406-49fb-8460-56702c381999)

**Comment by mcanouil** [comment]
You could possibly do that using JavaScript looking at `#fnref2` in the URL and changing the CSS of the corresponding element.
That's 100% HTML though. I'm not sure it fits as a Quarto feature, or it won't be in a near future in my opinion.

**Reply by mcanouil** [reply]
Something like:

```javascript
window.document.addEventListener("DOMContentLoaded", function (event) {
    document.addEventListener('click', function (e) {
        if (e.target.tagName === 'A' && e.target.getAttribute('href').startsWith('#')) {
            e.preventDefault();
            let id = e.target.getAttribute('href').substring(1);
            let element = document.getElementById(id);
            if (element) {
                element.style.border = '2px solid red';
                # element.scrollIntoView({ behavior: 'smooth', block: 'start' });
            }
        }
    });
});
```

It might conflict with other JavaScript already at play though.
It's a very rough draft.
